ignite/metrics/accuracy.py

```diff
@@ -98,7 +100,8 @@ class Accuracy(_BaseClassification):
     - `update` must receive output of the form `(y_pred, y)` or `{'y_pred': y_pred, 'y': y}`.
     - `y_pred` must be in the following shape (batch_size, num_categories, ...) or (batch_size, ...).
     - `y` must be in the following shape (batch_size, ...).
-    - `y` and `y_pred` must be in the following shape of (batch_size, num_categories, ...) for multilabel cases.
+    - `y` and `y_pred` must be in the following shape of (batch_size, num_categories, ...) and
+      num_categories must be greater than 1 for multilabel cases.
```

Comment:
I think it should produce wrong indentation. This piece of code should be merged with the previous sentence.

Reply:
I think this should work fine - please see attached screenshot.

![image](https://user-images.githubusercontent.com/17307490/84784384-167a5c00-af9f-11ea-992f-bfd929f9d971.png)

Reply:
That is perfect ! Sorry for the extra work...